### DISCUSSION TITLE
wrong relationship

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#2818] [Help] [answered]
Hello, 

I would like to resize %2.jpg, but

composite  %1.jpg %2.jpg -resize 10%% %3.jpg

resizes the composition of %1 and %2. 

How to change that?

**Comment by t246246** [comment, marked as answer]
I guess your %1 is not a filename itself but a parameter in windows batch file.
If so, simply use parenthesis around your target like ( %2.jpg -resize 10%% )

**Comment by fmw42** [comment]
You cannot do -resize in composite. You need to use convert or better mogrify, but one image at a time. Convert cannot have two new output names and mogrify will not let you rename files. So best to write a script loop and convert one image at a time.
